crates/viewer/re_space_view_spatial/src/contexts/transform_context.rs

```diff
@@ -385,26 +385,26 @@ fn transform_info_for_upward_propagation(
         reference_from_entity *= entity_from_2d_pinhole_content.inverse();
     }
 
-    // Collect & compute leaf transforms.
-    let (mut reference_from_instances, has_leaf_transforms) = if let Ok(mut entity_from_instances) =
-        SmallVec1::<[glam::Affine3A; 1]>::try_from_vec(
-            transforms_at_entity.entity_from_instance_leaf_transforms,
+    // Collect & compute poses.
+    let (mut reference_from_instances, has_instance_transforms) =
+        if let Ok(mut entity_from_instances) = SmallVec1::<[glam::Affine3A; 1]>::try_from_vec(
+            transforms_at_entity.entity_from_instance_pose_transforms,
```

Comment:
We're mixing words here - the archetype is called `InstancePoses3D`, so I suggest we follow that

```suggestion
    let (mut reference_from_instances, has_instance_poses) =
        if let Ok(mut entity_from_instances) = SmallVec1::<[glam::Affine3A; 1]>::try_from_vec(
            transforms_at_entity.entity_from_instance_poses,
```